packages/lib/src/vis/tiles/utils.ts

```diff
@@ -105,8 +101,34 @@ export function scaleToLayer(box: Box3, layerSize: Size, meshSize: Size): Box2 {
     clamp: true,
   });
 
+  return { xScale, yScale };
+}
+
+export function scaleToLayer(box: Box3, layerSize: Size, meshSize: Size): Box2 {
+  if (box.isEmpty()) {
+    return new Box2();
+  }
+
+  const { xScale, yScale } = getLayerScales(layerSize, meshSize);
+
   return new Box2(
     new Vector2(xScale(box.min.x), yScale(box.min.y)),
     new Vector2(xScale(box.max.x), yScale(box.max.y))
   );
 }
+
+export function scaleFromLayerToTile(layerCoord: Vector2, tileSize: Size) {
```

Comment:
New util needed for the tooltip: from the position of the mouse in `layer` coordinates (`layerCoord`), we want to know what is the underlying tile offset and the position of the mouse in this tile.

Reply:
Is this still used?